### ISSUE TITLE
Bug: Appeal charged to POA that isn't listed nor IHP writing

### ISSUE THREAD

**Issue by MeredithStewart** [department-of-veterans-affairs/caseflow#10234]
This appeal shows assigned to "National Law School Veterans Clinic Consortium," (a POA VSO) but it is believed that this an error:  

https://appeals.cf.ds.va.gov/queue/appeals/3191420 

From Sal 4/1/19: The hearing was successfully cancelled, no change in location in VACOLS but is now assigned to the National Law School Veterans Clinic consortium?  There is no POA for this appeal in VACOLS or VBMS.

Asked Dustin to look and he believes it is in error. He checked file to see if there's any documents for POA and there isn't. Doesn't seem to be any reason for it to be assigned to that particular

**Comment by mdbenjam** [comment]
The problem here is this user doesn't have a POA in BGS and we have a bunch of VSOs with nil for their participant ids. So we think anyone without representation has these VSOs as representation. Should have a fix soon.

**Comment by lowellrex** [comment]
The incorrect behaviour here was indeed solved by the fix @mdbenjam released in #10256.

```ruby
  rails c> a = LegacyAppeal.find_by(vacols_id: "3191420")
  rails c> sync_record = a.record_synced_by_job.find_by(sync_job_name: UpdateAppellantRepresentationJob.name)
  rails c> sync_record.attempted!
  rails c> appeal_new_task_count, appeal_closed_task_count = TrackVeteranTask.sync_tracking_tasks(a)
  # [0, 4]
  rails c> sync_record.processed!
  rails c> a.tasks.active.pluck(:type)
  # ["DispositionTask", "HearingTask", "RootTask"]
```

**Comment by MeredithStewart** [comment]
Closing, this will be updated when the job runs on these.
